### DISCUSSION TITLE
User Defined Functions with R

### DISCUSSION THREAD

**Discussion by ran-codes** [duckdb/duckdb#12438] [Ideas]
Our team uses the tidyverse API and some basics statistics function in R for much of our data work. We have shifted to Duckdb - particualrly the duckdb-dbt  workflow - and would be a gamechanger if we could do some of our R work in duckdb via UDF.

I have read https://github.com/duckdb/duckdb/issues/471 but am wonder if there is plans for support for other language UDFs. I think it would be a game changing feature.

**Comment by Tishj** [comment]
That's a question for <https://github.com/duckdb/duckdb-r> I think?
